e2e_tests/tests/experiment/experiment.py

```diff
@@ -442,25 +439,27 @@ def run_list_cli_tests(experiment_id: int) -> None:
     )
 
 
-def report_failed_experiment(experiment_id: int, state: str) -> None:
-    print(
-        "Experiment {} terminated in {} state unexpectedly!".format(experiment_id, state),
-        file=sys.stderr,
-    )
-
+def report_failed_experiment(experiment_id: int) -> None:
```

Comment:
would it be useful to add a target/current state parameters here to only dump the problematic trial's logs? not important since most tests only have 1 trial anyway.

Reply:
turns out that all the two places where this is called log that info already, each with useful context about where in the test the failure occurred.